library/src/interactor_impl.cxx

```diff
@@ -603,6 +826,73 @@ interactor& interactor_impl::setDropFilesCallBack(
   return *this;
 }
 
+//----------------------------------------------------------------------------
+interactor& interactor_impl::addCommandCallback(
+  std::string_view action, std::function<bool(const std::vector<std::string>&)> callback)
+{
+  this->Internals->CommandCallbacks[std::string(action)] = callback;
+  return *this;
+}
+
+//----------------------------------------------------------------------------
+interactor& interactor_impl::removeCommandCallback(std::string_view action)
+{
+  this->Internals->CommandCallbacks.erase(std::string(action));
+  return *this;
+}
```

Comment:
Just use a const ref string here, no reason to use a string_view
```suggestion
interactor& interactor_impl::removeCommandCallback(const std::string& action)
{
  this->Internals->CommandCallbacks.erase(action);
  return *this;
}
```

Reply:
this is C++11

Reply:
nevermind, you are right, this API should not use string_view I think.